scripts/ci/travis/tests/contenta_ci_tests_helper.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Setup Anonymous User Function
+# Gives access to anonymous user to access protected resources.
+# This function receives one argument:
+#   $1 -> The Drupal Base Path
+setup_anonymous_user() {
+     if [ -z $1 ] ; then
+        echo "Please pass the Contenta Project Base Path to the install_test_dependencies function " 1>&2
+        exit 1
+    fi
+
+    # Setup local variables
+    current_path=`pwd`
+    drush=$1/bin/drush
+    drupal_base=$1/web
+
+    cd $drupal_base
+    # Add Permission to anonymous user
+    $drush role-add-perm 'anonymous'  'access jsonapi resource list' -y
+    $drush updatedb -y
+    $drush cr -y
+
+    cd $current_path
+}
+
+# Install PHPUnit Composer Function
+# Install PHPUnit through Composer
+# This function receives one argument:
+#   $1 -> The Drupal Base Path
+install_phpunit_composer() {
+    if [ -z $1 ] ; then
+        echo "Please pass the Contenta Project Base Path to the install_phpunit_composer function " 1>&2
+        exit 1
+    fi
+
+    current_path=`pwd`
+    cd $1
+    composer require --dev phpunit/phpunit:~5.7
```

Comment:
This composer require install `phpunit` at the contenta-jsonapi-project level, and not at the contenta-jsonapi level. The `phpunit` in the `composer.json` is going to be removed. 

@dawehner do you think it will be proper to add `phpunit` as a dev dependency in contenta-json-api-project?